packages/manager/apps/telecom/src/app/telecom/pack/move/offers/move-offers.controller.js

```diff
@@ -108,6 +113,11 @@ export default class PackMoveOffersCtrl {
           );
         } else {
           this.listOffersFiber = fiberOffers.result.offers;
+          this.listOffersFiber.map((offer) => {
+            set(offer, 'displayedPrice', offer.prices.price.price);
```

Comment:
fyi the es6 way of doing this is :
```js
this.listOffersFiber.map((offer) => ({
  ...offer,
  displayedPrice: offer.prices.price.price,
  eligibilityReference: this.eligibilityReferenceFiber,
}));
```

Reply:
@Steffy29 , maybe this can be done in this PR as this change doesn't seem big and avoids having side effects by mutating an object

Reply:
@marie-j it's updated